FreestarAds-Ogury/Source/objc/OguryFSTRPartnerInitializer.m

```diff
@@ -0,0 +1,31 @@
+//
+//  OguryFSTRPartnerInitializer.m
+//  FreestarAds-Ogury
+//
+//  Created by Carlos Alcala on 7/04/22.
+//
+
+#import "OguryFSTRPartnerInitializer.h"
+#import <OgurySdk/Ogury.h>
+#import <OguryAds/OguryAds.h>
+#import <OguryChoiceManager/OguryChoiceManager.h>
+
+@implementation OguryFSTRPartnerInitializer
+
++ (instancetype)shared {
+    static OguryFSTRPartnerInitializer *instance = nil;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        instance = [[OguryFSTRPartnerInitializer alloc] init];
+    });
+    return instance;
+}
+
+- (void)runInitialization {
+    // run partner initialization here
+    FSTRLog(@"OGURY: Partner initialization.");
+    OguryConfigurationBuilder *configurationBuilder = [[OguryConfigurationBuilder alloc] initWithAssetKey:@"OGY-773308830772"];
+    [Ogury startWithConfiguration:[configurationBuilder build]];
```

Comment:
If you look at some of the other examples we have, the partner initializers need to inform back to the FSTRPartnerInitializerDelegate with initialization status.

Reply:
Will check